azure-pipelines.yml

```diff
@@ -212,6 +216,24 @@ stages:
               repositoryName: 'keboola/$(imageName)'
               pushTag: latest
 
+          - task: Docker@2
+            displayName: 'Push to GAR [$(productionTag)]'
+            inputs:
+              command: push
+              containerRegistry: 'Keboola GAR'
+              dockerfile: '$(Build.SourcesDirectory)/Dockerfile'
+              repository: '$(garRepositoryProjectId)/$(imageName)/$(imageName)'
+              tags: $(productionTag)
+
+          - task: Docker@2
```

Comment:
taky, latest by v produkci k ničemu neměl být a nepushoval bych ho. 

ALE
naopak budeme potřebovat pushnout latest v deploy_test, aby se dali na nějakým (dosud neexistujícím) GCP k8s clusteru spustit funkční testy.

Reply:
latest odstraním ale push z deploy_test by pak měl jít do nějakého jiné repository než do produkčního ne?

Reply:
JJ, však jsou tam jiný repositáře https://github.com/keboola/job-runner/pull/233/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5L134

Schválně jsem to napsal jako "budeme potřebovat". Nemusíš to dělat teď

Reply:
ok, to pak dořešíme v rámci jiných issue

Reply:
tak si to někam poznač